apps/escrow/frontend/src/systems/Core/routes.tsx

```diff
@@ -1,11 +1,10 @@
-import { Navigate, Route } from "react-router-dom";
+import { Route } from "react-router-dom";
 
-import { RequireWallet } from "./components/RequireWallet";
-import CreateWallet from "./pages/CreateWallet";
 import EscrowPage from "./pages/Escrow";
 import { Pages } from "./types/pages";
 
 export const coreRoutes = (
+  <Route path={Pages.escrow} element={<EscrowPage />} />
   <>
```

Comment:
This line causes `pnpm run dev` to fail so I cannot run the project

Reply:
hmm I'm not getting that error when I run `pnpm run dev` I'll look into to

Reply:
I probably messed up the merge from master